barretenberg/cpp/src/barretenberg/crypto/merkle_tree/lmdb_store/functions.cpp

```diff
@@ -0,0 +1,129 @@
+#include "barretenberg/crypto/merkle_tree/lmdb_store/functions.hpp"
+#include "barretenberg/numeric/uint256/uint256.hpp"
+#include <cstdint>
+#include <cstring>
+#include <functional>
+#include <vector>
+
+namespace bb::crypto::merkle_tree {
+void ThrowError(const std::string& errorString, int error)
+{
+    std::stringstream ss;
+    ss << errorString << ": " << error << " - " << mdb_strerror(error) << std::endl;
+    throw std::runtime_error(ss.str());
+}
+
+std::vector<uint8_t> SerialiseKey(uint8_t key)
```

Comment:
we don't use camelcase function names in C++ typically

Reply:
Should be fixed